sheetScript.gs

```diff
@@ -38,8 +57,8 @@ function doGet(e) {
 function findNextRow(data) { // Finds the row with the earliest last updated time
   var minTimestamp = data.slice(reservedTableRows).reduce(
     function (min, row, index) {
-      if (row[reservedColumnTimestamp].length != 0 && row[reservedColumnTimestamp] < min[1]) {
-        return [index, row[reservedColumnTimestamp]]
+      if (row[reservedColumnPlaylist].length != 0 && row[reservedColumnPlaylist] < min[1]) {
+        return [index, row[reservedColumnPlaylist]]
```

Comment:
Only the first index should be changed, the others are needed to compare timestamps to find the earliest one.

```suggestion
      if (row[reservedColumnPlaylist].length != 0 && row[reservedColumnTimestamp] < min[1]) {
        return [index, row[reservedColumnTimestamp]]
```